src/node/server/index.ts

```diff
@@ -123,8 +123,8 @@ export function createServer(config: ServerConfig): Server {
     if (optimizeDeps.auto !== false) {
       await require('../optimizer').optimizeDeps(config)
     }
-    context.port = port
-    return listen(port, ...args)
+    listen(port, ...args)
```

Comment:
We need to keep the same return value from `listen`.

Reply:
Understood. Can you please check the updated code. Open to the variable name feedback :)